common/src/main/java/tachyon/exception/InvalidArgumentException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.exception;
+
+/**
+ * Exception used when the client specified an invalid argument, indicating arguments that are
+ * problematic regardless of the state of the system. This differs from FailedPreconditionException,
+ * the latter is related to system state.
+ *
+ * For example:
+ * <ul>
+ * <li>malformed TachyonURL</li>
+ * <li>invalid tier alias(like -2)</li>
+ * <li>BlockStoreLocation does not belong to current tiered storage</li>
+ * </ul>
+ */
+public class InvalidArgumentException extends Exception {
```

Comment:
I wanted to use checked exception `InvalidArgumentException` instead of unchecked exception `java.lang.IllegalArgumentException`, but, yeah, who will recover from illegal arguments they passed into a method themselves? make sense, I'll change to `java.lang.IllegalArgumentException`.